dashboard/api/pkg/store/queries/get_job_input_relations.sql

```diff
@@ -0,0 +1,42 @@
+WITH job_inputs AS (
+    SELECT
+        id AS JobID,
+        input_element ->> 'StorageSource' as storage_source,
+        input_element ->> 'CID' AS input
+    FROM
+        job,
+        json_array_elements(jobdata::json -> 'Spec' -> 'inputs') AS input_element
+    WHERE input_element ->> 'StorageSource' = 'IPFS'
+),
+     job_outputs AS (
+         SELECT
+             node_states.JobID,
+             node_states.NodeID,
+             node_states.storage_source,
+             node_states.output_cid
+         FROM (
+                  SELECT
+                      id                                             AS JobID,
+                      key                                            AS NodeID,
+                      value -> 'Shards' -> '0' ->> 'State'           AS State,
+                      value -> 'Shards' -> '0' -> 'PublishedResults' AS PublishedResults,
+                      value -> 'Shards' -> '0' -> 'PublishedResults' ->> 'StorageSource' as storage_source,
```

Comment:
we are going to have a lot of fun when pushing the new job schema as we no longer have shards or state grouped by node. To avoid breaking the dashboard in the future, can we have these queries run on `v1beta1` only? We will lose data from the new schema until we update the query, but that is better than breaking the whole thing

Reply:
Yeah good idea.